### PR TITLE
Reduxify contractor resources

### LINKED ISSUE #437: Reduxify activities: contractor resources
Hook up the activities contractor resources section of the site to the application state so that it can easily be synced with the API.

### PR DESCRIPTION
**Pull request by mgwalker**
Hooks up the contractor resources section to redux state.  There's a pretty glaring style problem where the start/end date inputs overflow and cover up part of the cost inputs.  I can bang on it and see if I can get something that makes sense, but I was hoping I could convince @brendansudol to take a look at it for me.  😬

Closes #437

### This pull request changes...
- On the `/activities` page, the "contractor resources" section is now visible

### This pull request is ready to merge when...
- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- ~The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented~
- ~The change has been documented~
  - ~Associated OpenAPI documentation has been updated~

### This feature is done when...
- ~Design has approved the experience~
- ~Product has approved the experience~

### CODE REVIEW

File: web/src/reducers/activities.js [hunk 1]
```diff
@@ -53,6 +72,20 @@ const reducer = (state = initialState, action) => {
...
       };
     }
+    case ADD_ACTIVITY_CONTRACTOR_RESOURCE:
+      return u(
+        {
+          byId: {
+            [action.id]: {
+              contractorResources: contractors => [
+                ...contractors,
+                newContractorResource(contractors.length)
```
Comment (mgwalker) [inline web/src/reducers/activities.js]: I'll fix this after #448 is merged, so I can make sure they both work the same way.

**Comment by brendansudol** [comment]
Yep, I'll happily take a took at the styling issues :)

File: web/src/reducers/activities.js (hunk 1 above)
Comment (brendansudol) [reply web/src/reducers/activities.js]: ready for you

**Comment by brendansudol** [comment]
before updating this @mgwalker, can you take a look at https://github.com/18F/cms-hitech-apd/pull/456? (just so we can stay consistent)

**Pull request merged**